### PR TITLE
Feature/setup sort and filter

### PR DESCRIPTION
**Pull request by syntialai**

### CODE REVIEW

File: features/tailormade_search/src/main/res/layout/activity_search.xml [hunk 1]
```diff
@@ -1,116 +1,112 @@
...
+        </com.google.android.material.appbar.MaterialToolbar>
+    </com.google.android.material.appbar.AppBarLayout>
 
-        <ImageView
-            android:id="@+id/image_view_search_state"
+    <com.google.android.material.tabs.TabLayout
+        android:id="@+id/tab_layout_search"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/activity_search.xml:34]: can it added with tools:visibility?

File: features/tailormade_search/src/main/res/layout/activity_search.xml [hunk 2]
```diff
@@ -1,116 +1,112 @@
...
+        android:layout_height="match_parent"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/tab_layout_search">
+
+    </androidx.viewpager2.widget.ViewPager2>
+
+<!--    <androidx.fragment.app.FragmentContainerView-->
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/activity_search.xml:69]: put TODO

File: features/tailormade_search/src/main/res/layout/fragment_search_design_result.xml [hunk 3]
```diff
@@ -1,63 +1,65 @@
...
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent"
-                tools:ignore="ContentDescription" />
+        <ImageView
+            android:id="@+id/image_view_no_design_data_state"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/illustration_no_data"
+            android:visibility="gone"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/fragment_search_design_result.xml:32]: add tools:visibility="visible"

File: features/tailormade_search/src/main/res/layout/fragment_search_design_result.xml [hunk 4]
```diff
@@ -1,63 +1,65 @@
...
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/image_view_no_design_data_state" />
+        <TextView
+            android:id="@+id/text_view_no_design_data_state"
+            style="@style/TextAppearance.TailorMade.Headline2.BlackHigh"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_24"
+            android:text="@string/search_state"
+            android:visibility="gone"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/fragment_search_design_result.xml:47]: add tools:visibility="visible"

File: features/tailormade_search/src/main/res/layout/fragment_search_design_result.xml [hunk 5]
```diff
@@ -1,63 +1,65 @@
...
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/text_view_no_design_data_state" />
+        <TextView
+            android:id="@+id/text_view_no_design_data_description_state"
+            style="@style/TextAppearance.TailorMade.Body1.BlackMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_16"
+            android:text="@string/search_description_state"
+            android:visibility="gone"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/fragment_search_design_result.xml:61]: add tools:visibility="visible"

File: features/tailormade_search/src/main/res/layout/fragment_search_tailor_result.xml [hunk 6]
```diff
@@ -1,63 +1,65 @@
...
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent"
-                tools:ignore="ContentDescription" />
+        <ImageView
+            android:id="@+id/image_view_no_tailor_data_state"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/illustration_no_data"
+            android:visibility="gone"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/fragment_search_tailor_result.xml:32]: add tools:visibility="visible"

File: features/tailormade_search/src/main/res/layout/fragment_search_tailor_result.xml [hunk 7]
```diff
@@ -1,63 +1,65 @@
...
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/image_view_no_tailor_data_state" />
+        <TextView
+            android:id="@+id/text_view_no_tailor_data_state"
+            style="@style/TextAppearance.TailorMade.Headline2.BlackHigh"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_24"
+            android:text="@string/search_state"
+            android:visibility="gone"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/fragment_search_tailor_result.xml:47]: add tools:visibility="visible"

File: features/tailormade_search/src/main/res/layout/fragment_search_tailor_result.xml [hunk 8]
```diff
@@ -1,63 +1,65 @@
...
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/text_view_no_tailor_data_state" />
+        <TextView
+            android:id="@+id/text_view_no_tailor_data_description_state"
+            style="@style/TextAppearance.TailorMade.Body1.BlackMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_16"
+            android:text="@string/search_description_state"
+            android:visibility="gone"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/fragment_search_tailor_result.xml:61]: add tools:visibility="visible"

File: features/tailormade_search/src/main/res/layout/activity_search.xml (hunk 1 above)
Comment (syntialai) [reply features/tailormade_search/src/main/res/layout/activity_search.xml:34]: Yes kak

File: features/tailormade_search/src/main/res/layout/activity_search.xml [hunk 9]
```diff
@@ -1,119 +1,115 @@
...
-        android:layout_height="match_parent"
-        tools:context=".feature.search.view.SearchActivity">
-
-        <com.google.android.material.appbar.AppBarLayout
-            android:id="@+id/top_appbar_layout_search"
-            android:layout_width="@dimen/dp_0"
-            android:layout_height="wrap_content"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
```
Comment (blasiusneri) [inline features/tailormade_search/src/main/res/layout/activity_search.xml:17]: apply formatter

**Review by blasiusneri — APPROVED** [review]

**Pull request merged**